pkg/volume/cephfs/cephfs.go

```diff
@@ -89,62 +88,66 @@ func (plugin *cephfsPlugin) GetAccessModes() []v1.PersistentVolumeAccessMode {
 }
 
 func (plugin *cephfsPlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, _ volume.VolumeOptions) (volume.Mounter, error) {
-	cephvs, _, err := getVolumeSource(spec)
+	localSecretRef, secretRef, err := getSecretRef(spec)
```

Comment:
odd, not sure where my previous comment on this went... I'd rather see this be `getSecretNameNamespace(spec, podNamespace) (secretName string, secretNamespace string, err error)` and keep the fallback logic encapsulated in the `getSecretNameNamespace` function